crates/optimism/chainspec/src/lib.rs

```diff
@@ -368,8 +499,10 @@ mod tests {
 
     #[test]
     fn op_mainnet_forkids() {
+        let op_mainnet = OpChainSpecBuilder::optimism_mainnet().build();
+        let _ = op_mainnet.genesis_hash.set(OP_MAINNET.genesis_hash.get().copied().unwrap());
```

Comment:
should this not move inside the function body of `OpChainSpecBuilder::optimism_mainnet`?

Reply:
I was also thinking of doing it but it's not done in the `ChainSpecBuilder` for some reason, maybe an oversight? If so can add it in both

Reply:
makes sense, just leave it for now. would be great if you can open an issue for this.